### PR TITLE
Hotfix/4.4.0 py version fix

### PR DESCRIPTION
**Pull request by maciekswat**
Fixing code to work with python 3.9 and recent numpy versions

### CODE REVIEW

**Review by tjsego — CHANGES_REQUESTED** [review]
Minor cleanup here. Otherwise, looks great.

**Review by tjsego — CHANGES_REQUESTED** [review]
Looks good. Just needs previously raised issues resolved.

File: conda-recipes/versions.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-version: 4.4.0
+version: 4.5.0
```
[the remark this answers is not in the document]
Comment (maciekswat) [reply conda-recipes/versions.yaml]: Fixed

File: VERSION.txt [hunk 2]
```diff
@@ -1 +1 @@
-4.4.0
\ No newline at end of file
+4.5.0
```
[the remark this answers is not in the document]
Comment (maciekswat) [reply VERSION.txt]: Fixed

File: conda-recipes/versions.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
-version: 4.4.0
+version: 4.4.1
```
[the remark this answers is not in the document]
Comment (maciekswat) [reply conda-recipes/versions.yaml:1]: Fixed

**Review by tjsego — APPROVED** [review]

**Pull request merged**